<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnchantmentWidget</class>
 <widget class="QWidget" name="EnchantmentWidget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="type_label">
       <property name="text">
        <string>Effect Type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="typeComboBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="limit_label">
       <property name="text">
        <string>Limit</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="limitSpinBox">
       <property name="maximum">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="potency_label">
       <property name="text">
        <string>Potency</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="potencyComboBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="primary_label">
       <property name="text">
        <string>Primary Mod</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="primaryComboBox"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="secondary_label">
       <property name="text">
        <string>Secondary Mod</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="secondaryComboBox"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="timer_label">
       <property name="text">
        <string>Timer</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="timerSpinBox">
       <property name="maximum">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>Save and close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
